### PR TITLE
typofixes by https://github.com/vlajos/misspell_fixer

### PR DESCRIPTION
**Pull request by vlajos**
**Please check if the PR fulfills these requirements**
- [ I hope so.] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [ NA] Tests for the changes have been added (for bug fixes / features)
- [ NA] Docs have been added / updated (for bug fixes / features)

**What kind of change does this PR introduce?** (check one with "x")
```
[ ] Bugfix
[ ] Feature
[ X] Code style update (formatting, local variables)
[ X] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[ ] Other... Please describe:
```

**What is the current behavior?** (You can also link to an open issue here)

Some typos in the code, mostly in documentation blocks.

**What is the new behavior?**

The typos had been fixed.

**Does this PR introduce a breaking change?** (check one with "x")
```
[ ] Yes
[X ] No
```

If this PR contains a breaking change, please describe the impact and migration path for existing applications: ...

**Other information**:

 * I am a bit uncertain about the commit message format. Let me know please whether/how should I modify it...
 * And generally I am not too familiar with the rules, let me know please if anything should have to be amended.

### CODE REVIEW

**Comment by vlajos** [comment]
(I have not known this pullapprove system. Let me know please if splitting/reorganising the PR would make the story smoother.)

File: packages/compiler/src/expression_parser/parser.ts [hunk 1]
```diff
@@ -666,7 +666,7 @@ export class _ParseAST {
   }
 
   /**
-   * An identifier, a keyword, a string with an optional `-` inbetween.
+   * An identifier, a keyword, a string with an optional `-` between.
```
Comment (gkalpak) [inline packages/compiler/src/expression_parser/parser.ts]: Why not `in-between`?

**Review by gkalpak — APPROVED** [review]
I love this :heart:
Extra points if you found them by reading through all files 😛

File: packages/compiler/src/expression_parser/parser.ts (hunk 1 above)
Comment (vlajos) [inline packages/compiler/src/expression_parser/parser.ts]: I think you are right. I reverted this one.

**Comment by gkalpak** [comment]
An approval from @IgorMinar or @mhevery would cover all necessary groups.

**Comment by vlajos** [comment]
I rebased a few times to fix the conflicts. Can I do anything else, to make the story slicker?

**Comment by wardbell** [comment]
I rebased for you and force pushed that commit into your PR. That took care of the merge conflicts.
I'm reviewing the changes next while Travis grinds away.

**Review by wardbell — APPROVED** [review]
Terrific!  LGTM. Keep up the good work.

**Comment by wardbell** [comment]
Can't get this to pass CircleCI because it's trapped in v1.
Created new PR #17716 that contains this commit so you'll still be credited for your fine work.
That PR closes this one.

**Pull request closed without merging**